web/app/components/workflow/index.tsx

```diff
@@ -187,13 +186,10 @@ const Workflow: FC<WorkflowProps> = memo(({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  const { handleRefreshWorkflowDraft } = useWorkflowUpdate()
   const handleSyncWorkflowDraftWhenPageClose = useCallback(() => {
     if (document.visibilityState === 'hidden')
       syncWorkflowDraftWhenPageClose()
-    else if (document.visibilityState === 'visible')
```

Comment:
If you open this page again then it won't be able to update by removing these lines.